runtime/lua/vim/lsp/util.lua

```diff
@@ -1261,7 +1261,7 @@ function M.stylize_markdown(bufnr, contents, opts)
   -- when ft is nil, we get the ft from the regex match
   local matchers = {
     block = { nil, '```+([a-zA-Z0-9_]*)', '```+' },
-    pre = { '', '<pre>', '</pre>' },
+    pre = { nil, '<pre>([a-z]*)', '</pre>' },
```

Comment:
```suggestion
    pre = { nil, '<pre>([a-z0-9]*)', '</pre>' },
```
(this is the exact specification in the parser)

Reply:
ok, cool. updated...